tests/queries/0_stateless/02771_specify_query_multiple_times.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+$CLICKHOUSE_LOCAL --query "SELECT 101"
+$CLICKHOUSE_LOCAL --query "SELECT 202" --query "SELECT 303"
```

Comment:
`./clickhouse-local --query "select 1" --query "select 2;select 1;"` produces

```
1
2
1
```

even if option `--multiquery` is not passed. That's kind of unexpected, the help for --multiquery says "If specified, multiple queries separated by semicolons can be listed after --query.".

EDIT: It is even more strange:
- ./clickhouse-local --query "select 1; select 2" works (without `--multiquery`)
- ./clickhouse-client --query "select 1; select 2" throws

EDIT of the EDIT: Interesting, this behavior in clickhouse-local and clickhouse-client already existed before. So this PR didn't cause this. But I am not sure if this is a bug or desired, at least `clickhouse-local --help` currently shows wrong information about `--multiquery`. @Ziy1-Tan Totally optional, but it would be great if you could check briefly if clickhouse-local could behave in the same way as clickhouse-client.